src/PrestaShopBundle/Resources/config/routing/admin/_authorization.yml

```diff
@@ -0,0 +1,36 @@
+_admin_login:
+    path: /
+    methods: [GET,POST]
+    # This condition is needed to prevent triggering the login route when
```

Comment:
👍

Reply:
This seems like a complicated condition, maybe we should change this route path, or add a prefix for the whole authorization routing file in `routing/admin.yml` ?

Reply:
I can change the login route, but I would still need to redirect to that new route when visiting `/`, so this condition would still be needed

Reply:
if I change the path `/` to let's say `/login`, we would still want to redirect to `/login` when the merchant visits `example.com/admin123`, so I'd need the same complex condition to recognize that `/` is not the same as `/index.php?controller=something`.

Or maybe the authentication error handler could take care of it and redirect to login page when visiting `/`, because an authorization error would occur :thinking:

Reply:
@matks this is one of the last questions that you mentioned, what should be done with it?

Reply:
I think it's a good thing to have a dedicated `/login` route
Even outside of the page migration, it's better to have `/` => dashboard and `/login` => login

That will reschedule this issue (the `/` route is shared by legacy controllers and migrated dashboard page) for later, when we migrate the dashboard.

@rokaszygmantas Would this work ?
1) create dedicated `/login` route
2) when visitor hits `/` route then
- if he's logged in, everything is all right, process the request as usual
- if he's not logged in, redirect him to `/login`